raitools/README.md

```diff
@@ -0,0 +1,16 @@
+# Responsible AI Tools SDK for Python
+
+### This package has been tested with Python 3.6, 3.7 and 3.8
+
+The Responsible AI Tools SDK enables users to analyze their machine learning models in one API. Users will be able to analyze errors, explain the most important features, validate fairness, compute counterfactuals and run causal analysis in a single API.
+
```

Comment:
Maybe change to "using a single API" towards the end

Reply:
done!